src/CSharp.Mongo.Migration.Test/CSharp.Mongo.Migration.Test.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
@JordanDChappell I'm not across how this works in plugins like this, does this make it not usable by lower versions of .NET (such as by another project including this?)?

In the mobile world, speaking of Android more specifically you could specify a minimum framwork level and then a target framework level, I assume C# would have something like this to help with this scenario?

Maybe this should be removed from this PR and further explored?

Reply:
@yoshifp a change to this test project should be okay as it's not included in the published package that's uploaded to NuGet.

The library project is targetting .NET standard in this case which is often used for high compatibility purposes.